src/data/markdown/docs/02 javascript api/07 k6-experimental/02 redis/10 Client/Client-decrby.md

```diff
@@ -36,13 +36,18 @@ const redisClient = new redis.Client({
   password: redis_password,
 });
 
-export default function () {
-  redisClient
-    .set('mykey', 10, 0)
-    .then((_) => redisClient.incr('mykey'))
-    .then((value) => redisClient.incrBy('mykey', value))
-    .then((value) => redisClient.decrBy('mykey', value))
-    .then((_) => redisClient.decr('mykey'));
+export default async function () {
+    await redisClient.set('mykey', 10, 0);
+    
+    let value;
+    value = await redisClient.incr('mykey');
+    value = await redisClient.incrBy('mykey', value);
+    value = await redisClient.decrBy('mykey', value);
```

Comment:
I kind of feel like this example could've been a lot shorter and just used `set` and `decrBy` 

Currently we do a bunch of stuff with the value with different functions but not actually checking the result until the end.